modules/order/src/Plugin/views/area/OrderTotal.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Drupal\commerce_order\Plugin\views\area;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\views\Plugin\views\area\AreaPluginBase;
+use Drupal\views\Plugin\views\argument\NumericArgument;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Defines an order total area handler.
+ *
+ * Defines an order total area handler that shows the order total field with its
```

Comment:
"Defines an order total area handler that" is redudant, already written in the line above. Start with "Shows the order total...'